getPythonCmd.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Search for a compatible python version on the system or in the local path
+# and return the path to it if found, otherwise returns nothing
+
+set -euo pipefail
+
+PYTHON_VERSION=${1-}
+PYTHON_LOCAL_DIR=${2-}
+PYTHON_MAJOR_VERSION=${PYTHON_VERSION%.*}
```

Comment:
I'm just being pedantic here, but this is not the Major version. (It's Major.minor). I would either put a little comment to point it out (bash syntax for string manipulation is sometimes a little bit obscure), or have a better named variable.

Reply:
Good point